README.md

```diff
@@ -93,24 +95,20 @@ Consequently, it is recommended to set up two pools in your cluster:
 * A *worker* pool: any number of nodes with label `nodetype: worker` -- for workflow jobs. 
 This way jobs won't interfere with workflow runtime components.
 
-If you don't want to use labels, you can use branch `minikube` that doesn't have selectors.  
+If you don't want to use labels, you can use values from `minikube` directory that don't use selectors.  
 
 
-### Creating Kubernetes resources
-Create Kubernetes resources as follows:
+### Installing resources
+Assuming you are in repository main directory, install Kubernetes resources as follows:
 ```
-kubectl apply -f cm.yml
-kubectl apply -f nfs-server-service.yml
-kubectl apply -f redis-service.yml
-kubectl apply -f redis.yml
-kubectl apply -f nfs-server.yml
-kubectl apply -f pv-pvc.yml
-kubectl apply -f hyperflow-engine-deployment.yml
+helm install nfs-server charts/nfs-server --values values/cluster/nfs-server.yml
```

Comment:
nfs server should be a dependency of hyperflow-engine, also there is a chart that you can use here: 
https://github.com/onedata/charts/tree/master/stable/volume-nfs
get it's release from repo: helm repo add onedata https://onedata.github.io/charts/